challenge-2.js

```diff
@@ -0,0 +1,69 @@
+function convertRGBtoHex(...args) {
+  if (args.length !== 3) return "bad request!";
+  for (const arg of args) {
+    if (!isColorValid(arg)) return "bad request!";
+  }
+  let result = "#";
+  for (const arg of args) {
+    result += decToHex(arg);
+  }
+  return result;
+}
+
+const isColorValid = function (colorCode) {
```

Comment:
it is better to keep the consistency of creating a function by using either `function declaration` or `function expression`